Formula/s/snakeviz.rb

```diff
@@ -30,15 +31,20 @@ def install
 
   test do
     system "#{bin}/snakeviz", "--version"
+    system "python3", "--version"
+    system "python", "--version"
+    system "python3", "-m", "cProfile", "-o", "output.prof", "-m", "cProfile"
```

Comment:
There isn't a `python3` in `python@3.12` yet but `python3.12` will work here.

Reply:
Thanks. Test on Linux pass now.